anitya/api_v2.py

```diff
@@ -471,7 +475,11 @@ def post(self):
             "homepage": fields.Str(required=True),
             "backend": fields.Str(required=True),
             "version_url": fields.Str(missing=None),
+            "version_scheme": fields.Str(missing=None),
```

Comment:
```suggestion
            "version_scheme": fields.Str(missing="RPM"),
```
`version_scheme` should be always set, it's not required field in the API, but it's set to RPM in `create_project` function if not provided. Let's set it to "RPM" if it's not provided.

Reply:
Makes sense! Thanks for the review.